http_15.go

```diff
@@ -0,0 +1,16 @@
+// +build !go1.6
```

Comment:
Sexy, where did you find this in the docs? I couldn't figure it out. 

Also is this working with non-release versions (like tip for example)?